index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
```

Comment:
This page looks good to me! You've used semantically appropriate elements to lay out the page.